### ISSUE TITLE
pynx584 reports away when armed home

### ISSUE THREAD

**Issue by John5373** [kk7ds/pynx584#27]
I am using this with home assistant and it reports back as armed away despite being armed home. The code appears to report away vs home depending on whether a zone is bypassed. Is this correct? I don’t bypass any zones, just remove delays when armed stay.

**Comment by kk7ds** [comment]
Yep, from memory, that's right. The alarm system itself doesn't have a notion of "home" or "away" in terms of state, which means there's not really anything else stable to key on...
